hugolib/page.go

```diff
@@ -362,7 +363,16 @@ func (p *Page) ReadFrom(buf io.Reader) (int64, error) {
 }
 
 func (p *Page) analyzePage() {
-	p.WordCount = len(p.PlainWords())
+	p.WordCount = 0
+	for _, word := range p.PlainWords() {
+		runeCount := utf8.RuneCountInString(word)
+		if len(word) == runeCount {
+			p.WordCount++	
+		} else {
+			p.WordCount += runeCount
```

Comment:
@bep Yes, you are right. There is already a RuneCount method, so I reverted the change about WordCount. And I add RuneSummary method for those people who use Chinese or Janapese.
Ask a question, AppVeyor build failed randomly, do you know why?
Error message like that: 
Two or more menu items have the same name/identifier in "m1" Menu. Identified as "i2".